tools/kleaver/main.cpp

```diff
@@ -95,6 +95,11 @@ namespace {
   llvm::cl::opt<std::string> directoryToWriteQueryLogs("query-log-dir",llvm::cl::desc("The folder to write query logs to. Defaults is current working directory."),
 		                                               llvm::cl::init("."));
 
+  llvm::cl::opt<bool>
+  QueryIndependence("assume-query-independence",
```

Comment:
I like the original name better, and as long as the help message/documentation is clear about it, I don't think there's any confusion with the constraint independence optimization.